src/main/java/com/fundynamic/d2tm/game/behaviors/SelectableImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.fundynamic.d2tm.game.behaviors;
+
+import com.fundynamic.d2tm.game.map.MapEntity;
+import org.newdawn.slick.Color;
+import org.newdawn.slick.Graphics;
+
+public class SelectableImpl {
```

Comment:
Ah yes it should implement it. Had a hard time thinking of a good name.

On Sat, Feb 21, 2015 at 12:10 AM, Arjen van der Ende
<notifications@github.com> wrote:

>> @@ -0,0 +1,66 @@
>> +package com.fundynamic.d2tm.game.behaviors;
>> +
>> +import com.fundynamic.d2tm.game.map.MapEntity;
>> +import org.newdawn.slick.Color;
>> +import org.newdawn.slick.Graphics;
>> +
>> +public class SelectableImpl {
> Since we have a `Selectable` interface, I'd expect `SelectableImpl` to extend from it (based on the normal naming convention). So maybe rename this to `SelectionOutline` or something like that?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/stefanhendriks/dune2themaker4j/pull/42/files#r25111361